### ISSUE TITLE
How to use Vue components inside the markdown 

### ISSUE THREAD

**Issue by peoray** [ZTL-UwU/shadcn-docs-nuxt#35]
I'm building a docs site using this awesome template. I want to be able to preview the Vue code inside of the markdown file.

An example can be found [here](https://www.shadcn-vue.com/docs/components/accordion.html),  basically where we have preview and code. I checked the source code and saw they use a `ComponentPreview` component.

Is this possible in this template?

**Comment by ZTL-UwU** [comment]
Yes, this feature is planed. I'll implement it once I figure out a good DX.

Also, checkout [Inspira UI](https://github.com/unovue/inspira-ui) (if you haven't) which is also a Magic UI / Aceternity UI port to vue, and it uses shadcn-docs-nuxt as well.
